src/test/java/org/apache/commons/io/input/XmlStreamReaderTest.java

```diff
@@ -78,6 +82,26 @@ protected void _testRawNoBomInvalid(final String encoding) throws Exception {
         }
     }
 
+    @Test
+    protected void testNullFileInput() throws IOException {
+        assertThrows(NullPointerException.class, () -> new XmlStreamReader((File)null));
```

Comment:
Has commons-io decided to move  Java 8 minimum? Maven still requires Java 7 and could not upgrade to this.

Reply:
```xml
  <properties>
    <maven.compiler.source>1.8</maven.compiler.source>
    <maven.compiler.target>1.8</maven.compiler.target>
    <commons.componentid>io</commons.componentid>
    <commons.module.name>org.apache.commons.io</commons.module.name>
    <commons.rc.version>RC1</commons.rc.version>
    <commons.bc.version>2.6</commons.bc.version>
    <commons.release.version>2.7</commons.release.version>
    <commons.release.desc>(requires Java 8)</commons.release.desc>
```